source/tls_channel_handler.c

```diff
@@ -93,6 +120,7 @@ int aws_tls_ctx_options_init_client_mtls_from_path(
     options->allocator = allocator;
     options->max_fragment_size = g_aws_channel_max_fragment_size;
 
+    /* this code already handles null termination properly */
```

Comment:
This comment doesn't mean anything to the next person to read it unless you give more context.

Reply:
removed, it was unnecessary anyways.